snuba/datasets/deletion_settings.py

```diff
@@ -1,13 +1,14 @@
 from __future__ import annotations
 
-from dataclasses import dataclass
+from dataclasses import dataclass, field
 from typing import Sequence
 
-MAX_ROWS_TO_DELETE_DEFAULT = 1000
+MAX_ROWS_TO_DELETE_DEFAULT = 100000
```

Comment:
What happens if an issue has more than these many rows?

Reply:
its looks like this function https://github.com/getsentry/snuba/blob/55398e54eaa9d96c4ce6a3eeb6816e41195cb975/snuba/web/delete_query.py#L137
will raise an exception and the delete will fail.

@MeredithAnya reading this function I realize it could cause an issue. Since we enforce max rows and fail in delete_from_table. if there are multiple tables the first one delete could succeed, but the second has too many rows and fails causing inconsistency. I think we need to change it so we check max rows on all tables before we start deleting on any.

Reply:
I will hold off implementing the bug fix until this PR is merged since it would cause a merge conflict. I am happy to take on this change though.